scripts/inactive_issue_checker.py

```diff
@@ -488,3 +493,11 @@ def unassign_issues(self, issues: List[Issue]) -> None:
             logging.info('Unassignment is currently disabled.')
     else:
         logging.info('No inactive issues found that need unassignment.')
+
+
+if __name__ == '__main__': # pragma: no cover
+    # This installs third party libraries before importing
+    # other files or importing libraries that use
+    # the builtins python module (e.g. build, utils).
```

Comment:
Probably explicitly mention 'requests' here, since this script is otherwise independent of the main codebase (so the current command is a bit misleading)?